mkdocs.yml

```diff
@@ -37,13 +37,12 @@ plugins:
       handlers:
         python:
           selection:
-            docstring_style: "restructured-text"
+            docstring_style: "sphinx"
```

Comment:
This PR is indicated from the description to remove the custom `mkdocs` handler but I see it also updates the `docstring_style` to `sphinx`.

Is there any possible risk with making this change as part of this PR? Had we been planning to change from `restructured-text` to `sphinx`?

Reply:
That makes a lot of sense, thank you!